src/fexists.ts

```diff
@@ -10,12 +10,22 @@ export function fexists(
 	specifier: Specifier,
 ) {
 	const parentUrl = `${pathToFileURL(dirname(parentPath)).href}/`;
-	const resolvedSpecifier = URL.canParse(specifier)
-		? specifier
-		// import.meta.resolve here is required because we need node's resolution algorithm to
+	
+	let resolvedSpecifier: string;
```

Comment:
I know this ultimately resolves to the same `string` type, but `FSPath` is more descriptive (and not just any string is acceptable).

```suggestion
	let resolvedSpecifier: FSPath;
```